tools/google-cloud-support-slackbot/SupportCase.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
```

Comment:
Following the Google style guide files names should be all lower case https://www.python.org/dev/peps/pep-0008/#package-and-module-names.

Please rename files to low case with an undescore.

Reply:
Fixed!